ui/src/app/base/reducers/pod/pod.ts

```diff
@@ -1,7 +1,22 @@
 import { createStandardReducer } from "app/utils/redux";
 
 import { pod as podActions } from "app/base/actions";
-import { PodState, SelectPodAction } from "app/base/types";
+import { Pod, PodState, TSFixMe } from "app/base/types";
+
+type RefreshPodErrorAction = {
```

Comment:
Makes sense to me!

Reply:
Hmm actually it makes for an awkward import/export pattern. Since all the action files to this point just export a single object, we would have to either import like:
`import podActions, { ActionType1, ActionType2 }  from "app/base/actions/pod/pod";`

or update the pod index export to:
`export { default, ActionType1, ActionType2 } from "./pod";`

to be able to get:
`import podActions, { ActionType1, ActionType2 } from "app/base/actions/pod";`

Anyway it seems like we would have to remember to add a new action type in a few different places. Maybe this would be best to handle as part of restructuring the directory.